README.md

```diff
@@ -30,16 +30,50 @@ month: 2
 year: 2023
 ---
 
-your markdown content
+# This is a heading style one
+
+**This is a featured Paragraph it cannot contain bold text but can contain [links](https://www.unitary.fund).**
+
+More text
+
+::image[d394zg4utwqxynenamqf]
+
+## This is a heading in heading style two
+
+This is a standard paragraph with a [LINK](https://www.google.com). **This is bold paragraph text**
+
+- this is a list item
+- this is **bold** list item
+- this is a long list item which has lots of words in it and will probably break onto a second or even a third line so we can see how that looks at differnt sizes adn make sure it looks as it is intended too
+- this is a list item with a [LINK](https://www.google.com) or [two)](https://www.google.com) added in it. 
+
+### This is a heading in heading style three introducing a video
+
+https://www.youtube.com/watch?v=dB_3R84ewig
```

Comment:
I wonder if instead of demo-ing a subset of markdown here to act as a template, it might be more helpful to provide a link to more thorough documentation on how to use markdown.
- [GitHub docs](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)
- [Write the Docs](https://www.writethedocs.org/guide/writing/markdown/)
- There's even a [markdown tutorial](https://www.markdowntutorial.com/)

Reply:
For me it is useful to show an example, as there are some elements specific here  as far as I can tell, e.g., 
```
**This is a featured Paragraph it cannot contain bold text but can contain [links](https://www.unitary.fund).**
```
or 
```
::image[d394zg4utwqxynenamqf]
```
Even if standard markdown, these prove quite useful to me even if I spent time previously editing markdown, due to how the website works.

Also in favor of having a template file, but could iterate in a later version.